src/ipumspy/api/extract.py

```diff
@@ -69,6 +70,8 @@ class Variable(IpumsObject):
     """Attach characteristics specifications"""
     data_quality_flags: Optional[bool] = False
     """Flag to include the variable's associated data quality flags if they exist"""
+    adjust_monetary_values: Optional[bool] = False
```

Comment:
Is this actually Optional? The default provided is False

Hope all is well, Renae!!

Reply:
Hi Kevin! Doing well and hope you are the same.

Yes, this is actually optional. This flag works similarly to the `data_quality_flags` argument; setting this flag to True adds an additional variable to your extract that contains inflation-adjusted dollar values. Currently the 2010 is the dollar to which all values are standardized. Also similarly to the `data_quality_flags` arg, setting this flag to True when an inflation-adjusted dollar value is not possible will result in a `BadIpumsApiRequest` error.